rocketpy/Rocket.py

```diff
@@ -787,7 +785,7 @@ def addParachute(
         # Return self
         return self.parachutes[-1]
 
-    def setRailButtons(self, position, angularPosition=45):
+    def setRailButtons(self, position, angular_position=45):
```

Comment:
Due to the fact that the term `position` is used generally in RocketPy as a single value `float`, I would change this parameter's name to something more descriptive, such as `pair_position` or `rb_position`.